cases/index.html

```diff
@@ -1,431 +1,501 @@
+                        </li>
+                    </ul>
+                </div>
+                <h3 class="case__subheading">Quotes</h3>
+                <div class="case__quotes">
+                    <section class="customer-quote-section">
+                        <p class="quote-text">
+                            I have never worked with anyone who has taken on so much respon&shy;sibility.
+                        </p>
+                        <p class="quotee">Signe Trolle Gronemann, Specialkonsulent and Product Owner</p>
+                    </section>
+                </div>
+            </div>
+        </div>
+        <div class="case" id="nornorm">
+            <div class="case__head">
+                <div class="case__intro">
+                    <img
+                        src="/img/nornorm-km.png"
+                        class="case__logo"
+                    >
+                    <h2 class="case__teaser">
+                        Working with <strong>Kabell & Munk</strong>, we supplied a scalable force of developers for <strong>NORNORM</strong>, an
+                        <a
+                            href="https://nornorm.com/press-releases/nornorm-wins-the-nordic-proptech-awards-in-space-as-a-service/"
+                            target="_blank"
+                            >award-winning property technology startup</a
+                        >, helping with development of their core product offering.
+                    </h2>
+                    <p>
+                        We worked on building a generalized UI with reusable basic components, allowing stakeholders
+                        with no programming knowledge to assemble webpages more quickly. This allows for a faster
+                        iteration speed, and makes it easier to adjust to changing needs.
+                    </p>
+                    <ul>
+                        <li>
+                            <a href="https://kabellmunk.dk/" target="_blank">
+                                Kabell & Munk
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://nornorm.com/" target="_blank">
+                                NORNORM
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="case__graphic">
+                    <img
+                        class="case__image"
+                        src="case_nornorm_km.jpg"
+                        alt="nornorm and kabel & Munk case image"
+                    >
+                </div>
+            </div>
+            <div class="case__body">
+                <h3 class="case__subheading">Summary</h3>
+                <div class="case__summary">
+                    <ul>
+                        <li>
+                            We made our expertise available to prioritization decisions, working with Kabell & Munk to
```

Comment:
"We made our expertise available to prioritization decisions" does not make much sense to me

maybe "We made our expertise available on prioritizing decision making" is better? I'm not sure if "prioritize decision making" is the point we are trying to get across though?

@hypesystem @Arth101

Reply:
entire text:
> We made our expertise available to prioritization decisions, working with Kabell & Munk to
ensure that good technical decisions were made for the long term sustainability of the technical
platform.

Reply:
Et forslag:
> We reinforced the development team with our expertise on task prioritization and ensuring hat good technical decisions were made for the long term sustainability of the technical platform.

Reply:
I like it 👍